src/components/more-blogs-section/more-blogs-section.jsx

```diff
@@ -9,14 +9,25 @@ import { Card } from "../card";
 
 import * as styles from './more-blogs-section.module.scss';
 
-const MoreBlogsSection = ({ relatedPosts, refMoreFrom, title, category }) => {
+const MoreBlogsSection = ({ relatedPosts, refMoreFrom, title }) => {
+  const filters = [
```

Comment:
This got duplicated, right? why not define it just once and export it?

Reply:
Done